src/main/java/thefarm/orchards/apple/model/Apple.java

```diff
@@ -0,0 +1,88 @@
+package thefarm.orchards.apple.model;
+
+public class Apple {
+    private String stem;
+    private String blossom;
+    private String peeling;
```

Comment:
Does this describe the apple in the process of peeling while on the vine? Or the state of having been peeled or not? Is a String the best type for it?

Reply:
I guess this should be "peel" rather than "peeling"

Reply:
Would you use a peel attribute? Or would you use something like ``peeled`` or ``isPeeled``? Or are there other purposes for an apple peel?

Reply:
Although the peel is eatable, it is also the container of the apple. It protects the inner part of the apple and helps to retain the juice.

Reply:
In a production environment, what use would the description of the Apple's container/protection/juice-retention have that justifies the memory and CPU time taken by the extra code? ;)